critiquebrainz/ws/review/test/views_test.py

```diff
@@ -72,6 +72,61 @@ def test_review_delete(self):
         resp = self.client.delete('/review/%s' % review["id"], headers=self.header(self.user))
         self.assert200(resp)
 
+    def test_review_type(self):
+        review_type_all = db_review.create(
+            entity_id="1b3abc15-7453-39f3-86c4-1441f360e121",
+            entity_type='release_group',
+            user_id=self.user.id,
+            text="Testing! This text should be on the page.",
+            rating=5,
+            is_draft=False,
+            license_id=self.license["id"],
+        )
+        review_only_rating = db_review.create(
+            entity_id="2b3abc25-7453-39f3-86c4-1441f360e121",
+            entity_type='release_group',
+            user_id=self.user.id,
+            rating=5,
+            is_draft=False,
+            license_id=self.license["id"],
+        )
+        review_only_review = db_review.create(
+            entity_id="3b3abc35-7453-39f3-86c4-1441f360e121",
+            entity_type='release_group',
+            user_id=self.user.id,
+            text="Testing! This text should be on the page.",
+            is_draft=False,
+            license_id=self.license["id"],
+        )
+
+        response = self.client.get('/review/', query_string={'review_type': 'rating'})
+        self.assert200(response)
+        actual_review_ids = [review['id'] for review in response.json['reviews']]
+        expected_review_ids = [str(review_type_all['id']), str(review_only_rating['id'])]
+        self.assertCountEqual(actual_review_ids, expected_review_ids)
+
+        response = self.client.get('/review/', query_string={'review_type': 'review'})
+        self.assert200(response)
+        actual_review_ids = [review['id'] for review in response.json['reviews']]
+        expected_review_ids = [str(review_type_all['id']), str(review_only_review['id'])]
+        self.assertCountEqual(actual_review_ids, expected_review_ids)
+
+    def test_review_large_count(self):
```

Comment:
we should add a comment to explain what this is testing (that `count` is the total number of reviews with this given review_type, which may be different from the number of returned items on this page).
In order to be explicit it'd be good to add 1 rating review here, and check that without a param there are 101 reviews, and with review_type=review there are 100